### PR TITLE
Add `read_timeout` and `connect_timeout` properties to SproxydClient

### PR DESCRIPTION
**Pull request by JordanP**
These properties are useful if a user of sproxyd-client wants to know
the timeout values with which the SproxydClient was instantiated.

### CODE REVIEW

File: test/unit/test_sproxyd_client.py [hunk 1]
```diff
@@ -180,6 +186,7 @@ def test_get_url_for_object_no_endpoint_alive(self):
         self.assertRaises(SproxydException,
                           sproxyd_client.get_url_for_object, "")
 
+    @mock.patch('eventlet.spawn', mock.Mock())
```
Comment (JordanP) [inline test/unit/test_sproxyd_client.py]: This is not strictly related to this patch but it's still needed

File: scality_sproxyd_client/sproxyd_client.py [hunk 2]
```diff
@@ -41,28 +41,35 @@ class SproxydClient(object):
...
+        :param read_timeout: Proxy timeout
+        :type read_timeout: `float`
         :param logger: Logger used by methods on the instance
         :type logger: `logging.Logger`
         '''
 
         self._conn_timeout = conn_timeout
-        self._proxy_timeout = proxy_timeout
+
+        # `read_timeout` used to be named `proxy_timeout`. We maintain
```
Comment (NicolasT) [inline scality_sproxyd_client/sproxyd_client.py]: Do we need backwards compatibility for a library which hasn't had any release? Using full-fledged `**kwargs` feels a bit gross.

File: scality_sproxyd_client/sproxyd_client.py [hunk 3]
```diff
@@ -173,6 +180,14 @@ def has_alive_endpoints(self):
 
         return len(self._alive) > 0
 
+    @property
+    def conn_timeout(self):
```
Comment (NicolasT) [inline scality_sproxyd_client/sproxyd_client.py]: Docstring

File: scality_sproxyd_client/sproxyd_client.py [hunk 4]
```diff
@@ -173,6 +180,14 @@ def has_alive_endpoints(self):
...
+        return self._conn_timeout
+
+    @property
+    def read_timeout(self):
```
Comment (NicolasT) [inline scality_sproxyd_client/sproxyd_client.py]: Docstring

**Comment by NicolasT** [comment]
LGTM
